### PR TITLE
Tutorial revision

### PR DESCRIPTION
**Pull request by Ensamma**
Revision to tutorial. Will need up update links and image URLs.

### CODE REVIEW

File: Wiki/Software/Beginner-Windows.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+5. Plug the Leonardo into your computer.
+6. Press the reset button.
+
+<img src="https://github.com/Ensamma/Microcontroller/blob/master/Wiki/Software/images/tutorial-windows-leonardo-3.jpg" height="400">
+
+The QMK program will now flash the program to the Leonardo and show a bunch of logging.
+Afterwards, the 3 LEDs on the Leonardo should flash in unison for 5 seconds before turning off.
+
+<img src="https://github.com/Ensamma/Microcontroller/blob/master/Wiki/Software/images/tutorial-windows-leonardo-4.png" height="600">
```
Comment (Mysticial) [inline Wiki/Software/Beginner-Windows.md:73]: This image will need to be updated since the file name is for TurboA instead of PABotBase.

Comment (Ensamma) [reply Wiki/Software/Beginner-Windows.md:73]: Do we have a file for this? I'm not seeing it in the image folder for hardware or software.

Comment (Mysticial) [reply Wiki/Software/Beginner-Windows.md:73]: We don't. lol

File: Wiki/Software/Beginner-Windows.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
+# PART 4
```
Comment (Mysticial) [inline Wiki/Software/Beginner-Windows.md:1]: Where's Part 1-3 again?

**Pull request merged**